website/docs/components/form/primitives/partials/guidelines/guidelines.md

```diff
@@ -52,18 +54,11 @@ Don’t use helper text to communicate length requirements as this can result in
 
```

Comment:
Maybe there's some more nuance needed here, I think what I was trying to communicate initially is more along the lines of not presenting duplicative information between the two components. What Evan was outlining makes a lot of sense to me (specifically, seeing all of the requirements up front and having them persist), but we want to avoid having scenarios where both the helper text and character count are saying the same thing (more or less what is captured in the image).

Maybe it's as simple as softening this portion a bit and leaning a bit more on the duplicative nature, rather than being so strict. Couple of thoughts that might help:

- Avoid presenting duplicate information between the helper text and the character count.
- There is some intrinsic overlap between the two elements (they can both communicate the requirements of the field), but helper text is more of a "what" and character count is more of a progress indicator (maybe even a where/how? Bad example)

Reply:
```suggestion
Don't include length requirements in the helper text, as it may duplicate what is already indicated by the character count.
```

but also maybe adding in a DO block 

```suggestion
Include specific instructions other than length requirements in the helper text to complement rather than duplicate the information provided by the character count.
```
<img width="836" alt="character-count-do-helper-text" src="https://github.com/hashicorp/design-system/assets/124841193/e2b16fb4-4b0c-45a1-bef6-ccfed3ceec9a">

wdyt? @hashicorp/hds-design

Reply:
This "do" example relates more to helper text best practices than character count best practices. There is some basic guidance for helper text already, which we could beef up, but we still need to be careful about what we recommend. For example, I would avoid using language such as "other than length requirements" because that brings us back to the original issue where we're steering folks away from providing persistent information to the user in the helper text space (even if it is about length requirements). This "don't" language suffers from the same problem.

What Jory has outlined makes sense. You could keep the original screenshot but change the language to something like:

"Avoid presenting duplicate information between the helper text and the character count. Helper text should be used to provide persistent requirements while character count represents more of a progress indicator towards a length requirement."

Reply:
Ah okay, that makes sense. I'll make those changes